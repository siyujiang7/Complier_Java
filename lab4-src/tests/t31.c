long main() {
  return 0;
}

void P(double a, double b, char* c) {
  long m;
  m = "pizza" >= 2.3;
  return 1;
}

long F(char a) {
  a = g + 'b';
  return 2.3;
}

long G() {
  char* g;
  g = 2.3;
  return "dog";
}

long main() {
  long a;
  long g;
  long* m;
  char* lll;
  lll = -"dog";
  F();
  G('a');
  m["cat"] = 1 + a;
  a = 5;
  continue;
  break;
  long l;
  gg = 1;
  P("a", "b", 2.3);
  return 0;
}
