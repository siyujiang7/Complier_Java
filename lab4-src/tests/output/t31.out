----------
 Errors/Warnings: 17 
----------
1 : Code can't precede declarations
1 : Break not in loop
1 : Continue not in loop
1 : Returning value in void function
0 : Incompatible return type
1 : Incompatible type assigning
1 : Assigning to an undeclared variable
1 : Non-Integer array subscript
1 : Assigning pointer to integer
1 : Undeclared Variable in expression
0 : Divide by zero
1 : Invalid unary operation
1 : Invalid operands to binary operator
1 : Variable/Function Redeclaration
3 : Incompatible function argument types
1 : Too few function arguments
1 : Too many function arguments
